src/Command/Db/DbSizeCommand.php

```diff
@@ -377,12 +418,13 @@ private function getMySqlUsage(HostInterface $host, array $database) {
      *
      * @return string
      */
-    private function formatPercentage($percentage, $machineReadable) {
+    private function formatPercentage($percentage, $machineReadable, $blnUseColour=true) {
```

Comment:
Irrelevant / unnecessary change

Reply:
$blnUseColour : I use it, because we don't want to show colours when were simply listing the distribution of database usage size.
See: `platform db:size -p sudo27zgjpwqa -vvv -e WEBSUPPORT-170-office-renewal --relationship=management -D`